File: ios/GetSum.m
//
//  Torch.m
//  NativeModulesIntro
//
//  Created by Dipti Agrawal on 07/08/21.
//

#import "React/RCTBridgeModule.h"


@interface RCT_EXTERN_MODULE(GetSum, NSObject)
RCT_EXTERN_METHOD(returnSum:(NSInteger *)name num:(NSInteger)secondNum callback:(RCTResponseSenderBlock *)successCallback)

@end

File: ios/GetSum.swift
//
//  Torch.swift
//  NativeModulesIntro
//
//  Created by Dipti Agrawal on 07/08/21.
//

import Foundation

@objc(GetSum)
class getSum: NSObject{
 
  @objc
  func returnSum(_ firstNum: Int,num secondNum:Int,  callback successCallback: RCTResponseSenderBlock){
    NSLog("Log from Swift: \(secondNum) \(firstNum)")
       successCallback([NSNull(), firstNum+secondNum])
    }

}
